odxtools/multiplexer.py

```diff
@@ -20,15 +20,15 @@ class MultiplexerCase:
 
     short_name: str
     long_name: str
-    structure_ref: str
+    structure_ref: OdxLinkRef
     lower_limit: str
     upper_limit: str
 
     def __post_init__(self):
         self._structure: Optional[BasicStructure] = None
 
-    def _resolve_references(self, id_lookup: Dict[str, Any]) -> None:
-        self._structure = id_lookup.get(self.structure_ref)
+    def _resolve_references(self, odxlinks: OdxLinkDatabase) -> None:
+        self._structure = odxlinks.resolve(self.structure_ref)
         if self._structure is None:
             logger.warning(
```

Comment:
This warning will not longer trigger (dead code)

Maybe add `if self.structure_ref` ?